request-manager/graph/grapher.go

```diff
@@ -334,9 +356,11 @@ func newNoopNode(name, id string) *Node {
 	}
 }
 
-// Get the minimal set of job args that the sequence starts with.
-// In the context of a wider request of which this sequence is a part, there may be more job
-// args available, but this sequence should not access them, so they are irrelevant for our purposes.
+// getAllSequenceArgs returns the minimal set of job args that the sequence
+// starts with.
+// In the context of a wider request of which this sequence is a part, there may
+// be more job args available, but this sequence should not access them, so they
+// are irrelevant for our purposes.
 func getAllSequenceArgs(seq *spec.Sequence) map[string]bool {
```

Comment:
Little bit of a confusion here: comment says "minimal set" but the func name is "all".  The seems to return all args. Why, according to code comment, may there be more job args but this seq shouldn't access them?